src/OpenTelemetry.AutoInstrumentation/Tagging/Tags.cs

```diff
@@ -33,22 +33,6 @@ public static class Tags
     /// </summary>
     public const string Version = "version";
```

Comment:
I think that both `env`, and `version` can be removed. Together with `CommonTags` class.

Reply:
Good catch, removed.